src/main/org/bson/ByteBuf.java

```diff
@@ -306,6 +306,24 @@ public RubyFixnum getInt32() {
     return int32;
   }
 
+  /**
+   * Get a 32 bit integer from the buffer.
+   *
+   * @author Neil Shweky
+   * @since 2020.06.26
+   */
+  @JRubyMethod(name = "get_uint32")
+  public RubyFixnum getUInt32() {
+    ensureBsonRead();
+
+    long temp = this.buffer.getInt();
+    if (temp < 0) temp += Math.pow(2,32);
```

Comment:
Space after comma please, and use the block syntax with {} and the statement on a separate line please.